frontend/internal-packages/artifact/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@liam-hq/artifact",
+  "description": "Artifact types and utilities for Liam ERD",
+  "keywords": [
+    "artifact",
+    "erd",
+    "types"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/liam-hq/liam",
+    "directory": "frontend/internal-packages/artifact"
+  },
+  "license": "Apache-2.0",
+  "version": "0.0.0",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
Do not make it public. Please refer to other internal-packages.
A fix in another PR is fine.